packages/mux-player/src/index.ts

```diff
@@ -361,6 +361,17 @@ class MuxPlayerElement extends VideoApiElement {
 
     // toggles activeCues for a particular track depending on whether the user is active or not
     const toggleLines = (track: TextTrack, userInactive: boolean) => {
+      // if we're live, exit early from here because our controls design means captions don't need to shift
+      // unless secondary color is set, in which case the entire bottom control bar may be opaque
+      if (
+        this.streamType &&
+        ['live', 'll-live'].includes(this.streamType) &&
```

Comment:
nit (blocking): Let's use the constants from playback-core here (have a followup to de-dup all the stuff we no longer need now that you gave us external modules)